tool/site/client/src/components/prescription-summary/prescriptionSummaryView.tsx

```diff
@@ -144,7 +150,8 @@ export function createSummaryPrescriptionViewProps(
     onPageChange,
     editMode,
     setEditMode,
-    errors: {}
+    errors: {},
+    handleDownload: () => undefined
```

Comment:
rather than returning this dummy function could we just make the `handleDownload` prop optional?

Reply:
alternatively the return type here could be `Omit<PrescriptionSummaryViewProps, "handleDownload">`